test/text/vtt_text_parser_unit.js

```diff
@@ -586,6 +586,117 @@ describe('VttTextParser', () => {
   });
 
+  it('supports payload stylized', () => {
+    verifyHelper(
+        [
+          {
+            startTime: 10,
+            endTime: 20,
+            payload: '',
+            nestedCues: [
+              {
+                startTime: 10,
+                endTime: 20,
+                payload: 'Test',
+                fontWeight: Cue.fontWeight.BOLD,
+              },
+            ],
+          },
+          {
+            startTime: 20,
+            endTime: 30,
+            payload: '',
+            nestedCues: [
+              {
+                startTime: 20,
+                endTime: 30,
+                payload: 'Test2',
+                fontStyle: Cue.fontStyle.ITALIC,
+              },
+            ],
+          },
+          {
+            startTime: 30,
+            endTime: 40,
+            payload: '',
+            nestedCues: [
+              {
+                startTime: 30,
+                endTime: 40,
+                payload: 'Test3',
+                textDecoration: [Cue.textDecoration.UNDERLINE],
+              },
+            ],
+          },
+          {
+            startTime: 40,
+            endTime: 50,
+            payload: '',
+            nestedCues: [
+              {
+                startTime: 40,
+                endTime: 50,
+                payload: 'Test4',
+              },
+            ],
+          },
+          {
+            startTime: 50,
+            endTime: 60,
+            payload: '',
+            nestedCues: [
+              {
+                startTime: 50,
+                endTime: 60,
+                payload: 'Test',
+                fontWeight: Cue.fontWeight.BOLD,
+                fontStyle: Cue.fontStyle.NORMAL,
+              },
+              {
+                startTime: 50,
+                endTime: 60,
+                payload: '5',
+                fontWeight: Cue.fontWeight.BOLD,
+                fontStyle: Cue.fontStyle.ITALIC,
+              },
+            ],
+          },
+          {
+            startTime: 70,
+            endTime: 80,
+            payload: '',
+            nestedCues: [
+              {
+                startTime: 70,
+                endTime: 80,
+                payload: 'Test',
+                fontWeight: Cue.fontWeight.NORMAL,
+              },
+              {
+                startTime: 70,
+                endTime: 80,
+                payload: '6',
+                fontWeight: Cue.fontWeight.BOLD,
+              },
+            ],
+          },
+        ],
+        'WEBVTT\n\n' +
+        '00:00:10.000 --> 00:00:20.000\n' +
+        '<b>Test</b>\n\n' +
+        '00:00:20.000 --> 00:00:30.000\n' +
+        '<i>Test2</i>\n\n' +
+        '00:00:30.000 --> 00:00:40.000\n' +
+        '<u>Test3</u>\n\n' +
+        '00:00:40.000 --> 00:00:50.000\n' +
+        '<a>Test4</a>\n\n' +
+        '00:00:50.000 --> 00:01:00.000\n' +
+        '<b>Test<i>5</i></b>\n\n' +
+        '00:01:10.000 --> 00:01:20.000\n' +
+        'Test<b>6</b>',
```

Comment:
Would you please add one additional case?  We just clarified the rules and fixed the displayers with regard to whitespace.  Cue payloads should always contain exactly the whitespace that they want to be shown.  So something like this would be useful to verify that the VTT parser keeps internal whitespace inside nested cues:

```
<b>Test <i>7</i></b>
```

Which should create the payloads `'Test '` (with trailing space) and `'7'`.

Thanks!

Reply:
I added more tests